package.json

```diff
@@ -13,6 +13,8 @@
 		"alert",
 		"collapse",
 		"components",
+		"headless",
```

Comment:
If you want to add the tag `lifeless` to our headless packages, update the script [here](https://github.com/AmadeusITGroup/AgnosUI/blob/main/scripts/buildPackageJson.js#L34)